frontend/src/pages/ProgressionChecker/GridView/GridView.jsx

```diff
@@ -80,9 +90,25 @@ const GridView = ({ isLoading, structure }) => {
                   <Title level={2}>{subGroup}</Title>
                   <div className="courseGroup">
                     {subGroupEntry.map((course) => (
-                      <Button className="checkerButton" type="primary" style={course.completed ? null : { background: "#FFF", color: "#9254de" }} key={course.key}>
-                        {course.key}: {course.title}
-                      </Button>
+                      course.unplanned ? (
+                        <Badge
+                          count={(
+                            <Tooltip title="Course has not been planned">
+                              <div className="unplannedBadge">
+                                Unplanned
```

Comment:
Maybe this could be replaced by an exclamation icon rather than displaying text?